### PR TITLE
CS-1349-UpdatePortalReadme

### PR DESCRIPTION
**Pull request by Chazzysb**
CS-1349-UpdatePortalReadme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,82 @@ This repo should be used as a template when creating new repos for the Apprentic
...
+      "GoogleAnalytics": {
+        "GoogleTagManagerId": "GTM-MP5RSWL"
+      },
+      "ApprenticePortalOuterApi": {
+        "ApiBaseUrl": "https://localhost:5123/",
+        "SubscriptionKey": "key",
+        "ApiVersion": "1"
+      },
+      "ApplicationUrls": {
+        //"ApprenticeHomeUrl": "https://portal.at-aas.apprenticeships.education.gov.uk/",
```
Comment (benagain) [inline README.md:78]: It's worth removing the commented lines

File: README.md [hunk 2]
```diff
@@ -11,16 +11,82 @@ This repo should be used as a template when creating new repos for the Apprentic
...
+        //"ApprenticeLoginUrl": "https://accounts.at-aas.apprenticeships.education.gov.uk/",
+        "ApprenticeHomeUrl": "https://localhost:44398",
+        "ApprenticeCommitmentsUrl": "https://localhost:7070",
+        "ApprenticeLoginUrl": "https://localhost:5001",
+        "ApprenticeAccountsUrl": "https://localhost:7080"
+      },
+      "ZenDesk": {
+        "ZenDeskSnippetKey": "e0730bdd-a32c-4c39-8032-7d7a908eacb4",
+        "ZenDeskSectionId": "360003002699",
+        "ZenDeskCobrowsingSnippetKey": "Qec2OgXsUy8lsUO1iOuleFAMjl7AcIEGdOojlQjLesShBHx0wrM4xXlR7X1h9bEo"
```
Comment (benagain) [inline README.md:90]: I think we should replace the values here with dummy ones

File: README.md (hunk 1 above)
Comment (Chazzysb) [reply README.md]: These were left is as a dev may want to point to AT and not local.. removed now

File: README.md (hunk 2 above)
Comment (Chazzysb) [reply README.md]: done

**Review by benagain — APPROVED** [review]

**Pull request merged**